### PR TITLE
moved symlink creation to end of execution to allow checking for empt…

### PR DESCRIPTION
**Pull request by FrederickDeny**

### CODE REVIEW

File: e4s_cl/logger.py [hunk 1]
```diff
@@ -451,6 +452,22 @@ def add_file_handler(
...
+            os.unlink(LOG_LATEST)
+        except OSError as err:
+            _ROOT_LOGGER.debug("Unlink %s failed: %s", LOG_LATEST.as_posix(),
+                               str(err))
+
+        try:
+            os.symlink(Path(LOG_FILE.parent, LOG_ID), LOG_LATEST)
+        except OSError as err:
+            _ROOT_LOGGER.debug("Symlink %s failed: %s", LOG_LATEST.as_posix(),
+                               str(err))
```
Comment (spoutn1k) [inline e4s_cl/logger.py:467]: Merge both those blocks for readability

File: e4s_cl/logger.py [hunk 2]
```diff
@@ -451,6 +452,22 @@ def add_file_handler(
...
+        else:
+            os.environ[LOG_ID_MARKER] = LOG_LATEST.name
```
Comment (spoutn1k) [inline e4s_cl/logger.py:469]: Delete this as it becomes irrelevant when moved to the end of execution

**Review by spoutn1k — CHANGES_REQUESTED** [review]

File: e4s_cl/logger.py (hunk 2 above)
Comment (FrederickDeny) [reply e4s_cl/logger.py]: Done

File: e4s_cl/logger.py (hunk 1 above)
Comment (FrederickDeny) [reply e4s_cl/logger.py]: Done

**Review by spoutn1k — APPROVED** [review]

**Pull request merged**